recipe/meta.yaml

```diff
@@ -24,6 +24,7 @@ requirements:
     - numpy >=1.23.1
     - pandas >=1.4.3
     - sqlalchemy >=1.4.27
+    - polars
```

Comment:
@windiana42 @finn-rudolph this doesn't match upstream pyproject.toml https://github.com/pydiverse/pydiverse.transform/blob/0.2.1/pyproject.toml

we should update everything there as well s.t. basic `import pydantic.transform` still works